demo/api/src/db/migrations/Migration20220127092750.ts

```diff
@@ -3,7 +3,7 @@ import { Migration } from "@mikro-orm/migrations";
 export class Migration20220127092750 extends Migration {
     async up(): Promise<void> {
         this.addSql(
-            'create table "Page" ("id" uuid not null, "content" json not null, "seo" json not null, "createdAt" timestamp with time zone not null, "updatedAt" timestamp with time zone not null);',
+            'create table "Page" ("id" uuid not null, "content" json not null, "stage" json not null, "seo" json not null, "createdAt" timestamp with time zone not null, "updatedAt" timestamp with time zone not null);',
```

Comment:
```suggestion
            'create table "Page" ("id" uuid not null, "content" json not null, "seo" json not null, "createdAt" timestamp with time zone not null, "updatedAt" timestamp with time zone not null);',
```

This is done in the migration below.